.github/workflows/run-k8s-tests-on-arm64.yaml

```diff
@@ -0,0 +1,83 @@
+name: CI | Run kubernetes tests on arm64
+on:
+  workflow_call:
+    inputs:
+      registry:
+        required: true
+        type: string
+      repo:
+        required: true
+        type: string
+      tag:
+        required: true
+        type: string
+      pr-number:
+        required: true
+        type: string
+      commit-hash:
+        required: false
+        type: string
+      target-branch:
+        required: false
+        type: string
+        default: ""
+
+jobs:
+  run-k8s-tests-on-arm64:
+    strategy:
+      fail-fast: false
+      matrix:
+        vmm:
+          - qemu
+        k8s:
+          - kubeadm
+    runs-on: arm64-k8s
```

Comment:
Can you add `arm64-k8s` to the self-hosted runners list: https://github.com/kata-containers/kata-containers/blob/16aa6b9b4bb14e9b88db26d82db52904a17880c2/.github/actionlint.yaml#L9 please

Reply:
Done, thanks for pointing this out!